Xamarin.PropertyEditing.Mac/Controls/TimeSpanEditorControl.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections;
+using AppKit;
+using Xamarin.PropertyEditing.Mac.Resources;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	public class TimeSpanValidationTextDelegate : IValidationTextDelegate
+	{
+		public bool IsValid (NSText textObject) => TimeSpan.TryParse (textObject.Value, out var result);
```

Comment:
How is this handling localized input?

Reply:
AFAIK if is not set uses the CultureInfo.CurrentCulture of the thread. We do the same in the old GTK Editor. https://github.com/mono/monodevelop/blob/master/main/src/core/MonoDevelop.Ide/MonoDevelop.Components.PropertyGrid.Editors/TimeSpanEditor.cs#L84-L87

Reply:
Can you check whatever we're doing for floating point types, so we're consistent?

Reply:
Of course sir!